weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1496,7 +1405,7 @@ def _end_call_for_insert_to_ch_insertable_end_call(
         id=end_call.id,
         exception=end_call.exception,
         ended_at=end_call.ended_at,
-        summary_dump=_dict_value_to_dump(end_call.summary),
+        summary_dump=_dict_value_to_dump(dict(end_call.summary)),
```

Comment:
curious what happens with the nested typed dicts? is `dict` enough to fix this? Should this be implemented in `_dict_value_to_dump`?

Reply:
im going to move this to _any_value_to_dump and I confirmed this works for at least 5 deep nested dicts